pkg/platform/kube/getter.go

```diff
@@ -45,25 +45,27 @@ func (g *getter) get(consumer *consumer, getOptions *platform.GetOptions) ([]pla
 	functions := []platform.Function{}
 	functioncrInstances := []functioncr.Function{}
 
-	// if identifier specified, we need to get a single function
-	if getOptions.Name != "" {
+	// check if identifier specified, if so take identifiers and act accordingly
+	if len(getOptions.MatchCriterias) > 0 {
```

Comment:
The cobra `command/get.go` appends one wildcard matchCriteria if no arguments are given - this means the `else` of this `if` is never reached. However, I am pretty sure that functioncrClient.Get will not work with the wildcard matchCriteria (as it only returns one functioncr instance).

In short, we might need another more well-defined way to distinguish between `get` with no function name specifiers and `get` with them - the behavior shouldn't differ based on the platform.

The adaptation _may_ happen only here. For instance, if we only got only one matchCriteria and its Name is `""`, move to the else